packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1452,6 +1459,16 @@ export const components: ThemeOptions["components"] = {
         ...(ownerState.stickyHeader && {
           borderCollapse: "separate",
         }),
+
+        caption: {
+          clip: "rect(0 0 0 0)",
+          clipPath: "inset(50%)",
+          height: "1px",
+          overflow: "hidden",
+          position: "absolute",
+          whiteSpace: "nowrap",
+          width: "1px",
```

Comment:
What's with the 1x1 px height and width?

Reply:
Thank you!

Checkbox: was pre-flight also changed to preflight in the Behavior and Checked examples? (I see Indeterminate in the PR). Now will check Radio.

Reply:
Same question for Radio; I can see that the Hint text was updated from Lightspeed to Light speed but check if it was also done for the Default, Disabled, and Invalid examples. I may just not be scrolling properly!

Reply:
And, finally on Data Table: Agree with the description/caption change to `Information about the largest and smallest planets.` instead of `Plan interplanetary travel.`

I don’t know how to read the code change for the number, but believe it adds the comma. Thank you!